### ISSUE TITLE
Project Status? Successor?

### ISSUE THREAD

**Issue by ajmas** [MacDownApp/macdown#1335]
I see the project hasn't been updated since 2021, so should we consider MacDown to be abandoned at this point? If so, which project is the successor to MacDown?

**Comment by fuesec** [comment]
I think it is safe to assume that the project has been abandoned

**Comment by mickeyl** [comment]
Is there any promising fork? https://github.com/lucy-jane/macdown seems to be relatively current.

**Comment by 5T33Z0** [comment]
> Is there any promising fork? https://github.com/lucy-jane/macdown seems to be relatively current.

That fork contains the last pull request made to the main repo, but it doen't have any builds and unfortunately, I don't know how to compile this from source. Macdown is my favorite markdown editor and I really would love if someone would pick it up and continue the work.

<blockquote><img src="https://opengraph.githubassets.com/b88d4a20a064400b9ffa59a673337eaf081fde86a32ac7d699dfd22d6f5b829e/lucy-jane/macdown" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/lucy-jane/macdown">GitHub - lucy-jane/macdown: Open source Markdown editor for macOS.</a></strong></div><div>Open source Markdown editor for macOS. Contribute to lucy-jane/macdown development by creating an account on GitHub.</div></blockquote>

**Comment by finalyards** [comment]
Willing to do monetary support, if someone does. i.e. Kofi level

**Comment by 5T33Z0** [comment]
There's a new fork with a new release but it's only for Apple Silicon: https://github.com/tikkal/macdown-arm64 

<blockquote><img src="https://opengraph.githubassets.com/011019e02754a415186b8d518606d7b951fa7ff5f0e9e67ff098e41064ca1492/tikkal/macdown-arm64" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/tikkal/macdown-arm64">GitHub - tikkal/macdown-arm64: Open source Markdown editor for macOS, for Mac Silicon (arm64)</a></strong></div><div>Open source Markdown editor for macOS, for Mac Silicon (arm64) - tikkal/macdown-arm64</div></blockquote>

**Comment by schuyler** [comment]
Hi, all. For those of you are interested in Markdown editing, I’d like to share the pre-release of [MacDown 3000](https://macdown.app/), a reboot of the open source Markdown editor we all know and love. I am planning to keep up a cadence of semi-regular releases focusing on bug fixes and stability. We need beta testers. Please [download the app](https://macdown.app/release), give it a try, and [share your feedback](https://macdown.app/issues)!

**Comment by 5T33Z0** [comment]
@schuyler Awseome. I just tested it on macOS Sonoma (Intel). As far as I can tell, the most annoying issues from the old version have been fixed. No more screen "blinking" of the preview pane when editing, the 1:1 view is split up correctly, and the "Open Recent" menu works again as well. Thank you so much!

**Comment by schuyler** [comment]
@5T33Z0 can I quote that on the website as a testimonial? 😁

**Comment by 5T33Z0** [comment]
Yes, of couse. But I noticed that the 1:1 view still glitches out sometimes so that the screen is not really split 50/50, instead it's unevenly splitted.

29.11.2025 04:26:11 Schuyler Erle ***@***.***>:

**Comment by schuyler** [comment]
Ok! Please file an issue in the new project and we’ll see about fixing
that. A screenshot would be helpful too, if you can reproduce it. Thanks!
